text/083-boost-stdlib-dev/text.md

```diff
@@ -0,0 +1,786 @@
+    Sets/Multiset.v, Sets/Uniset.v, Sets/Relations_2.v, Sets/Image.v,
+    Sets/Relations_3.v, Sets/Relations_1.v, Sets/Classical_sets.v,
+    Sets/Finite_sets.v, Sets/Permut.v, Sets/Integers.v,
+    Sets/Finite_sets_facts.v, Sets/Powerset_facts.v, Sets/Powerset.v,
+    Sets/Powerset_Classical_facts.v, Sets/Ensembles.v,
+    Sets/Relations_1_facts.v, Sets/Constructive_sets.v,
+    Sets/Infinite_sets.v, Sets/Relations_2_facts.v,
+    Sets/Relations_3_facts.v, Sets/Cpo.v</details>
+* coq-roots-sorting
+  * The Coq standard library, sorting algorithms
+  * Axiomatizations of sorts. Namespace `Coq.Sorting`.
+  * 10 files, 4.1 kloc
+  * <details><summary>contains:</summary> Sorting/CPermutation.v,
+    Sorting/Permutation.v, Lists/SetoidList.v, Sorting/Sorting.v,
+    Sorting/Heap.v, Sorting/PermutEq.v, Lists/SetoidPermutation.v,
+    Sorting/Mergesort.v, Sorting/PermutSetoid.v,
+    Sorting/Sorted.v</details>
+* coq-roots-orders-ex
+  * The Coq standard library, sorting algorithms
+  * Instances of order structures from
+    coq-roots-structures. Namespace `Coq.Structures`.
+  * 10 files, 2.5 kloc
+  * <details><summary>contains:</summary> Bool/BoolOrder.v,
+    Structures/OrdersEx.v, Structures/OrdersLists.v,
+    Structures/EqualitiesFacts.v, Structures/OrderedTypeAlt.v,
+    Structures/DecidableType.v, Structures/OrderedTypeEx.v,
+    Structures/OrdersAlt.v, Structures/DecidableTypeEx.v,
+    Structures/OrderedType.v</details>
+* coq-roots-set-map
+  * The Coq standard library, finite sets and maps
+  * Modular implementation of finite sets/maps using lists or
+    efficient trees. Namespaces `Coq.FSets` and `Coq.MSets`.
+  * 34 files, 27.3 kloc
+  * <details><summary>contains:</summary> FSets/FMapInterface.v,
+    FSets/FSetInterface.v, FSets/FSetWeakList.v, FSets/FSetAVL.v,
+    FSets/FSetCompat.v, FSets/FSetDecide.v, FSets/FSetList.v,
+    FSets/FSetProperties.v, FSets/FMapList.v, FSets/FMapFacts.v,
+    FSets/FSetPositive.v, FSets/FMapFullAVL.v,
+    FSets/FSetToFiniteSet.v, FSets/FMapAVL.v, FSets/FMapPositive.v,
+    FSets/FSets.v, FSets/FMaps.v, FSets/FSetFacts.v,
+    FSets/FSetEqProperties.v, FSets/FSetBridge.v,
+    FSets/FMapWeakList.v, MSets/MSetList.v, MSets/MSetDecide.v,
+    MSets/MSets.v, MSets/MSetWeakList.v, MSets/MSetGenTree.v,
+    MSets/MSetAVL.v, MSets/MSetInterface.v, MSets/MSetFacts.v,
+    MSets/MSetProperties.v, MSets/MSetRBT.v, MSets/MSetPositive.v,
+    MSets/MSetToFiniteSet.v, MSets/MSetEqProperties.v</details>
+* coq-roots-compat
+  * The Coq standard library, compatibility files
+  * Some compatibility files with older versions. Namespace `Coq.Compat`.
+  * 5 files, 0.1 kloc
+  * <details><summary>contains:</summary> Compat/AdmitAxiom.v,
+    Compat/Coq817.v, Compat/Coq819.v, Compat/Coq818.v,
+    Compat/Coq820.v</details>
+* coq-roots-derive
+  * The Coq standard library, derive
+  * Namespace `Coq.derive`
+  * 1 file, 0.0 kloc
+  * <details><summary>contains:</summary>
+    theories/derive/Derive.v</details>
```

Comment:
In any case, we should refrain putting ML code into the library. But since it has no dependencies and is a single line file, it could remain in the stdlib.